packages/terra-graphs/src/components/Line/LineGraph.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint import/no-unresolved: off */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+import './LineGraph.module.scss';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+
+const propTypes = {
+  /**
+   * ID for graph created.
+   */
+  graphID: PropTypes.string.isRequired,
+  /**
+   * Graph configuration object
+   */
+  graphConfig: PropTypes.object.isRequired,
+  /**
+   * data to be displayed in graph
+   */
+  dataset: PropTypes.array,
```

Comment:
It would be helpful to use PropTypes.arrayOf to further define the individual item shapes.

Whole bunch of examples here: https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/106/commits/8f1b7d01eb1f006f648f922b7b0ca25095b52fb2#diff-21099d22317b65a81c95ece3f22955d83ab8a2ede2ccbde8f2c32ab64cb8b1eaR21).